CT-1571 always use our version of log/parallel_runtime_rspec.log

misaka opened the pull request:
We use this file so that Travis can balance out the tests between processes
better and so that they run more quickly. However, because it changes every time
we run the tests, this causes merge conflicts ... this change allows us to avoid
conflicts by using a merge strategy that will always choose our local version.
To work fully, developers should run this command to ensure the merge driver
defined in this file exists:

    $ git config --local merge.ours.driver true

EdwardAndress commented:
All done @aliuk2012

aliuk2012 commented on spec/features/03_respond_to_drafter_assignment/respond_to_assignment_spec.rb:
```diff
@@ -2,6 +2,13 @@
+
+# =======
+
+# >>>>>>> 53a631c... Ct-403 accept or reject button in case detail
```
Too many comments ;)

aliuk2012 commented on spec/features/03_respond_to_drafter_assignment/respond_to_assignment_spec.rb:
```diff
@@ -10,6 +17,8 @@
   end
 
+  #given(:drafter)         { kase.drafter }
+
```
More comments ;)

EdwardAndress commented:
Done, properly.

aliuk2012 commented on spec/features/mark_as_sent_spec.rb:
```diff
@@ -0,0 +1,9 @@
+    
+
+  end
+end
```
Should this file be in this PR?

EdwardAndress replied:
No, it certainly should not, the sneaky little..!

EdwardAndress commented:
I'll merge this once the tests pass - I had to resolve a merge conflict.

misaka committed:
CT-1571 always use our version of log/parallel_runtime_rspec.log

We use this file so that Travis can balance out the tests between processes
better and so that they run more quickly. However, because it changes every time
we run the tests, this causes merge conflicts ... this change allows us to avoid
conflicts by using a merge strategy that will always choose our local version.
To work fully, developers should run this command to ensure the merge driver
defined in this file exists:

  $ git config --local merge.ours.driver true
